lib/MwbExporter/Formatter/Doctrine2/Annotation/Model/Table.php

```diff
@@ -800,6 +811,27 @@ protected function writeColumnsGetterAndSetter(WriterInterface $writer)
         }
     }
 
+    protected function getApiPlatformAnnotations() {
+
+        $classes = [
+            OnCorps\ApiPlatformSortAnnotations::class => $this->parseComment('sort'),
+            OnCorps\ApiPlatformSearchAnnotations::class => $this->parseComment('search'),
```

Comment:
can we use CustomComment class to store `sort` and `search` as constant and then call this from there ?
thinking of that class was keeping a easily accessible list of what are our "custom tag" in mysql workbench which, if called as constant can be also changed in a second moment

Reply:
Of course - I'll make the change.